x-pack/solutions/security/plugins/security_solution/docs/testing/test_plans/detection_response/prebuilt_rules/upgrade.md

```diff
@@ -114,82 +116,68 @@ Status: `in progress`. The current test plan matches [Rule Immutability/Customiz
 
 ### Functional requirements
 
-- User should be able to install prebuilt rules with and without previewing what exactly they would install (rule properties).
 - User should be able to upgrade prebuilt rules with and without previewing what updates they would apply (rule properties of target rule versions).
-- If user chooses to preview a prebuilt rule to be installed/upgraded, we currently show this preview in a flyout.
-- In the prebuilt rule preview a tab that doesn't have any sections should not be displayed and a section that doesn't have any properties also should not be displayed.
+- User should be able to upgrade prebuilt rules with and without customizations. Where the following fields support customizations
 
-Examples of rule properties we show in the prebuilt rule preview flyout:
+```Gherkin
+Examples:
+│ Rule type         │ Field name in UI                  │ Diffable rule field     │
```

Comment:
`Diffable rule field` Why is it important to specify technical details of our domain model and API contract in a test plan? Do we elaborate on what these are and how are used?

Reply:
I'm going to address this in a follow-up PR.